debian/test.sh

```diff
@@ -11,37 +11,73 @@ configSetup
 
 cd ${DEBIAN_RELEASE}
 
+# Array variable storing built packages
+DEBS=(*.deb)
+
 ### Package Quality ###
 echo "Generating lintian logs"
 LINTFLAGS="-I --show-overrides"
-lintian ${LINTFLAGS} ${DEB} > ../cache/deb-lint.log
-lintian ${LINTFLAGS} ${DSC} > ../cache/dsc-lint.log
-lintian ${LINTFLAGS} ${CHANGES} > ../cache/changes-lint.log
-
-# Information about package
-echo "Generating package information log"
-lesspipe ${DEB} > ../cache/pkg-info.log
 
-set -x
+echo
+echo lintian ${LINTFLAGS} ${DSC}
+rm -f ../cache/dsc-lint.log
+lintian ${LINTFLAGS} ${DSC} | tee ../cache/dsc-lint.log
+
+echo
+echo lintian ${LINTFLAGS} ${CHANGES}
+rm -f ../cache/changes-lint.log
+lintian ${LINTFLAGS} ${CHANGES} | tee ../cache/changes-lint.log
+
+for DEB in ${DEBS[*]}
+do
+  BASE=`echo $DEB | cut -f 1 -d _`
+  echo
+  echo lintian ${LINTFLAGS} ${DEB}
+  rm -f ../cache/${BASE}-lint.log
+  lintian ${LINTFLAGS} ${DEB} | tee ../cache/${BASE}-lint.log
+done
+
+echo
+
+for DEB in ${DEBS[*]}
+do
+  BASE=`echo $DEB | cut -f 1 -d _`
+  echo
+ 
+  # Information about package
+  echo "Generating package information log for ${DEB}"
+  echo lesspipe ${DEB}
+  lesspipe ${DEB} > ../cache/${BASE}-pkg-info.log
+
+done
+
+#set -x
 
 # Install package
-sudo dpkg -i ${DEB}
+sudo dpkg -i ${DEBS[*]}
 
 which ${BINARY}
 
 ${BINARY} --version
 
-echo "Package contents:"
-ls -R /usr/lib/chapel > ../cache/pkg-contents.log
+echo "Summarizing Package contents:"
+ls -R /usr/share/chapel /usr/lib/chapel > ../cache/pkg-contents.log
 
 ls -l /usr/bin/${BINARY} >> ../cache/pkg-contents.log
 
-set +x
-
 echo "Tests finished."
 
 # Uninstall package (optionally)
-echo "Uninstall ${PKG}? (ctrl-c to cancel)"
+
+DEB_PKGS=""
+for DEB in ${DEBS[*]}
+do
+  BASE=`echo $DEB | cut -f 1 -d _`
+  DEB_PKGS="$DEB_PKGS $BASE"
+done
+
+echo "Uninstall $DEB_PKGS? (ctrl-c to cancel)"
 read response
 
-sudo dpkg -r ${PKG}
+echo sudo dpkg -r $DEB_PKGS
```

Comment:
nope, because I'm trying to expand muptiple thingns. DEB_PGKS contains a bunch of filenames (space separated)